internal/fnruntime/container.go

```diff
@@ -178,8 +181,23 @@ func (f *ContainerFn) prepareImage() error {
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 	cmd = exec.CommandContext(ctx, dockerBin, args...)
-	if _, err := cmd.CombinedOutput(); err != nil {
-		return fmt.Errorf("function image %q doesn't exist", f.Image)
+	if output, err := cmd.CombinedOutput(); err != nil {
+		return &DockerImageError{
+			Image:  f.Image,
+			Output: string(output),
+		}
 	}
 	return nil
 }
+
+// DockerImageError is an error type which will be returned when
+// the container run time cannot verify docker image.
+type DockerImageError struct {
+	Image  string
+	Output string
+}
+
+func (e *DockerImageError) Error() string {
+	return fmt.Sprintf(`Function image %q doesn't exist.
+Please verify the correctness and existence of the image name.`, e.Image)
```

Comment:
I think just saying `Function image %q does not exist` should be enough. second line doesn't help much to the end user :)

Reply:
done